Jenkinsfile

```diff
@@ -39,11 +40,6 @@ pipeline {
 
         // Docker build flags are set via the getDockerBuildFlags() shared library.
         sh "docker build ${getDockerBuildFlags()} -t rightbrainnetworks/auto-semver:${env.VERSION} ."
-
-
-        sh "python setup.py sdist"
- 
-        stash includes: "dist/semver-${env.SEMVER_NEW_VERSION}.tar.gz", name: 'PACKAGE' 
```

Comment:
If you dont stash you may not have access to the package in the other stages.

Reply:
Within Docker image